doc/classes/MultiMeshInstance2D.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance2D" inherits="Node2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances a [MultiMesh] in 2D.
+		Node that instantiates a [MultiMesh] in 2D.
```

Comment:
This should be "instances". In the Multimesh individual units are called "instances" and you don't instantiate them. Same goes with the change below